protocol/app/app.go

```diff
@@ -1232,9 +1261,6 @@ func New(
 	}
 	app.initializeRateLimiters()
 
-	app.ScopedIBCKeeper = scopedIBCKeeper
-	app.ScopedTransferKeeper = scopedTransferKeeper
```

Comment:
Potentially silly question, why remove these lines? Is it to not expose the IBC Keeper?

Reply:
Yup we removed `ScopedIBCKeeper` and `ScopedTransferKeeper` public fields. What I can tell this is conventionally done for testing purpose. However this is not utilized in our code. Removing these fields also prevent potential error where we pass un-initialized `app.ScopedIBCKeeper` into the `ibckeeper.NewKeeper()` function. Let me know if you still have question/concern!